AdyenUI/Styles/Buttons/AdyenButtonStyles.swift

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import UIKit
+
+/// Defines styles for different button types.
+public struct AdyenButtonStyles {
+
+    public var primary: AdyenButtonStyle
```

Comment:
Same as here, are they meant to be accessible/settable as direct properties? Generic approach is that they are not meant to be, which is why we have the init and/or the builder methods to modify them